src/common/types/cpuArchitecture.ts

```diff
@@ -14,12 +18,19 @@ export enum OcmCpuArchitecture {
 
 export type SupportedCpuArchitecture = Extract<
   CpuArchitecture,
-  CpuArchitecture.x86 | CpuArchitecture.ARM
+  CpuArchitecture.x86 | CpuArchitecture.ARM | CpuArchitecture.ppc64le | CpuArchitecture.s390x
 >;
 
-export const getSupportedCpuArchitectures = (): SupportedCpuArchitecture[] => [
-  CpuArchitecture.x86,
-  CpuArchitecture.ARM,
-];
+export const getSupportedCpuArchitectures = (
+  versions?: OpenshiftVersionOptionType[],
+  selectedVersionValue?: string,
+): CpuArchitecture[] => {
+  if (versions) {
+    const selectedVersion = versions.find((version) => version.value === selectedVersionValue);
+    return selectedVersion?.cpuArchitectures
```

Comment:
So all of the users will see the new CPU architectures, or only some of them should be able to see them?

Reply:
No, only users with CapabilityBareMetalInstallerMultiarch capability: https://issues.redhat.com/browse/MGMT-13600.
I can use the same condition that here: https://gitlab.cee.redhat.com/service/uhc-portal/-/blob/master/src/components/clusters/common/clusterStates.js#L119